network/src/commonMain/kotlin/com/wire/kalium/network/api/user/self/SelfUserInfoResponse.kt

```diff
@@ -55,5 +56,7 @@ enum class ManagedBy {
     }
 }
 
+@Deprecated("used for legacy compatibility")
```

Comment:
Does it get serialized as an empty object?

Is it because the `register` endpoint requires at least an empty object? If so, could you please clarify a bit and add details here?

Reply:
Is swagger `picture` is a list of `Pict` which is an empty object. So, I implemented it as it is in the swagger.
Maybe we are better off deleting it, since it's optional and the `assets` has the user picture.

Reply:
But I think it's a legacy thing where the `Picture` is a Base64 encoded image. (just speculation)